test/filterEvents.test.ts

```diff
@@ -0,0 +1,52 @@
+import { describe, expect, test } from '@jest/globals'
+import app from '../src/app'
+import supertest from 'supertest'
+import build from '../src/db/build'
+import sequelize from '../src/db/connection'
+
+beforeEach(() => build())
+
+describe('Filter events tests', () => {
+  test('length of all products', (done) => {
+    supertest(app)
+      .get('/api/v1/events')
+      .expect('Content-Type', /json/)
+      .end((err, res) => {
+        if (err) return done(err)
+        expect(res.body.length).toEqual(5)
+        return done()
+      })
+  })
+
+  test('get all events filtered by status:closed', (done) => {
+    supertest(app)
+      .get('/api/v1/events/?status=closed')
+      .expect('Content-Type', /json/)
+      .end((err, res) => {
+        if (err) return done(err)
+        console.log(res.body)
+
+        expect(res.body[0].description).toEqual(
+          'you should wear a custom'
+        )
```

Comment:
you are testing the description of the first event

why you did not tested the status of the event ?